composer.json

```diff
@@ -1,24 +1,58 @@
 {
   "name": "drupal/cloudflare",
-  "type": "module",
+  "type": "drupal-module",
   "description": "Drupal module for interacting with CloudFlare's SDK. CloudFlare is a copyright of CloudFlare, Inc.  The authors of this tool has no association with CloudFlare, Inc.",
-  "keywords": ["Drupal, CloudFlare", "Cloud Flare", "CDN"],
+  "keywords": ["Drupal", "CloudFlare", "Cloud Flare", "CDN"],
   "homepage": "https://github.com/d8-contrib-modules/cloudflare",
   "license": "GPL-2.0+",
   "minimum-stability": "dev",
+  "prefer-stable": true,
+  "repositories": [
+    {
+      "type": "composer",
+      "url":  "https://packagist.drupal-composer.org"
+    }
+  ],
+  "extra": {
```

Comment:
We don;'t need this extra for a module AFAIK, at least themes and profiles b/c we are not dependent on any of